### DISCUSSION TITLE
How do I add a newline to the comments field in Linux?

### DISCUSSION THREAD

**Discussion by RobertBernstein** [beetbox/beets#4635] [Q&A] [answered]
How do I insert a newline in the comments field in Linux? In Windows with PowerShell I can run the following:

`beet modify whitesnake comments="Purchased on CD.<backtick>nFrom Amazon"` where \<backtick\> is the backtick character.  The comments field will look like this:

     Purchased on CD.
     From Amazon

I tried using `\n` on Linux and it doesn't work.  Please help.

**Comment by sampsyo** [comment, marked as answer]
Hi! Can you explain more about what _does_ happen when you try this on Linux?

Broadly, this is more of a question about your (Linux) shell than it is about beets. When you write something like `beet modify foo="bar baz"`, beets never sees those quotes—they are only there to tell your *shell* how to parse the command line. Beets sees the list of argument strings `modify` and `foo=bar baz`. I'm not sure how your shell is interpreting the literal character `\n`, but perhaps you can try typing an actual newline character there.

**Reply by RobertBernstein** [reply]
Typing an actual newline character worked! I had no idea the shell (/bin/bash) would actually split the line if I hadn't yet typed the second quote.  Thanks for the help!  (Sorry for the delayed response.)

**Comment by JOJ0** [comment]
@RobertBernstein have you tried wrapping your string into single quotes? On most Linux shells this will prevent any interpretation of special characters of the shell itself. It could help, but could also lead to just having a literal `\n` character in your final string,  which is not what you want. Give it a try though! :-)

Also try what @sampsyo pointed out already, open your string with a double quote and just hit enter at the position where you want the newline and close the string with another double quote character. Usually shells understand that.

**Reply by RobertBernstein** [reply]
Thanks for recommending I try @sampsyo's solution.  That worked!  And thanks for your patience with my response.
